verify.go

```diff
@@ -641,23 +641,27 @@ func (d *App) verifyECRImage(ctx context.Context, image, region string) error {
 }
 
 func (d *App) verifyRegistryImage(ctx context.Context, image, user, password string) error {
-	rr := strings.SplitN(image, ":", 2)
-	image = rr[0]
-	var tag string
-	if len(rr) == 1 {
-		tag = "latest"
+	var imageName string
```

Comment:
In distribution/reference, the part before tag/digest is called name, so I named that part `imageName`. If you have any opinions, I will revise it.

https://github.com/distribution/reference/blob/727f80d42224f6696b8e1ad16b06aadf2c6b833b/reference.go#L6